packages/json-extension/src/component.tsx

```diff
@@ -38,6 +38,8 @@ function getStyle(tag: Tag): string {
   return jupyterHighlightStyle.style([tag]) ?? '';
 }
 
+StyleModule.mount(document, jupyterHighlightStyle.module as StyleModule);
```

Comment:
This is not a good practice as it will mount the stylesheet even if the component is not used. So you should do this in the `componentDidMount` of `Component` (to be added):

```typescript
componentDidMount(): void {
  StyleModule.mount(document, jupyterHighlightStyle.module as StyleModule);
}
```

See [documentation](https://reactjs.org/docs/state-and-lifecycle.html#adding-lifecycle-methods-to-a-class) if you want to know more about React component life cycle

Reply:
Ah okay sorry I missed that. Thanks for the reference to the docs. Will change.